charts/Util.ts

```diff
@@ -255,6 +255,11 @@ export function precisionRound(num: number, precision: number) {
     return Math.round(num * factor) / factor
 }
 
+export function roundSigFig(num: number, sigfigs: number = 1) {
+    const magnitude = Math.floor(Math.log10(Math.abs(num)))
```

Comment:
Actually just noticed that `Math.log10(0)` is `-Infinity`, so we might also want a case for good ol' zero.

Reply:
Good catch!